src/common/color.tokens.json

```diff
@@ -57,6 +57,16 @@
           }
         }
       },
+      "error": {
+        "value": "#ba2121",
+        "comment": "The highlight color for error states and/or messages.",
+        "$extensions": {
+          "dte.metadata": {
+            "isCurated": true,
+            "category": "color"
+          }
+        }
+      },
```

Comment:
sorry, the ticket was badly specified by Joeri, but we don't need this design token, instead we want to make sure that the error state in our theme is properly configured for the content component: `--of-formio-content-error-border-inline-start-color` which is the primary design token. The fallback to `--of-color-error` is for legacy/backwards compatibility. So I think we need to add `src/components/formio-content.tokens.json` and in there define `of > formio-content > error > border-inline-start-color` and point to `{of.color.danger}`, because that token already exists.

The relevant story state is https://open-formulieren.github.io/open-forms-sdk/?path=/story/form-io-components-vanilla-content--content&args=customClass:error